ai-platform-unified/notebooks/unofficial/pipelines/pipelines_intro_kfp.ipynb

```diff
@@ -878,14 +878,14 @@
       "source": [
         "### Create a service account (quick start version)\n",
         "\n",
-        "See the documentation for more details on the process of creating and configuring a service account to work with AI Platform Pipelines. In brief: \n",
+        "See the documentation for more details on the process of creating and configuring a service account to work with Vertex AI Pipelines. In brief: \n",
         "\n",
         "- Go to the Cloud Console -> IAM & Admin -> Service Accounts, click “**CREATE SERVICE ACCOUNT**”. \n",
```

Comment:
In PR #341 , I changed this section (which was no longer correct) and just swapped in a link to the docs.  So there may be a merge conflict here. 😬 (sorry).
I don't _think_ any of my other changes on that PR will cause a conflict, but something to watch out for.

Reply:
I may check your PR and replicate the changes in this PR to fix it.

Reply:
For pipelines, I saw conflict but just used your changes as I assume you already switched out the names/links.